includes/class-gp-route-translation-helpers.php

```diff
@@ -35,9 +35,15 @@ public function __construct() {
 	 * @return void
 	 */
 	public function discussions_dashboard( $locale_slug ) {
-		if ( ! is_user_logged_in() ) {
-			$this->die_with_404();
+		$roles_adapter = null;
```

Comment:
I think ideally we'd use something like `GP::$permission->current_user_can( 'approve'` something (maybe you can look up other usages in the code base to find how to identify GTEs)